golem-worker-service/src/worker_bridge_request_executor.rs

```diff
@@ -92,7 +92,7 @@ mod internal {
             .invoke_and_await_function_json(
                 &worker_id.into_target_worker_id(),
                 worker_request_params.idempotency_key,
-                worker_request_params.function_name.to_string(),
```

Comment:
Before, we had a string, which we then parsed  it to `ParsedFunctionName` and then converted back to `String` to call the invoke function in worker-service. A redundant roundtrip, and here we just avoid it.

Reply:
Making it elegant would make every function receive a nice `ParsedFunctionName` instead of `String` but then has 2 parsing and display call involved in hot path.

I haven't gone deeper into what if invoke function accepts a ParsedFunctionName until it reach worker-executor. But that's not the concern of this PR. Also `String` would be optimal especially there is a GRPC call and I can see why it is the way it is now.